examples/langchain_examples/pyproject.toml

```diff
@@ -9,6 +9,8 @@ license = "MIT"
 package-mode = false
 requires-python = ">=3.10,<3.13"
 dependencies = [
+    "pangea-multipass",
+    "pangea-multipass-langchain",
```

Comment:
We should add version here and remove `pangea-multipass` as it should be already included in `pangea-multipass-langchain`